packages/asc-core/src/components/HeaderStyle/HeaderTitleStyle.ts

```diff
@@ -0,0 +1,41 @@
+import styled from '../../styled-components'
+import { getBreakpointFromTheme } from '../../utils'
+
+const HeaderTitleStyle = styled.div<{}>`
+  @media screen and ${({ theme }) =>
+    getBreakpointFromTheme(theme, 'max-width', 'tablet')} {
+    flex-grow: 1;
+  }
+
+  & > h1 {
+    display: flex;
+    height: 100%
+    margin: 0;
+    background-color: rgba(192, 192, 192, 0.1);
+  }
+
+  & > h1 > span {
+    display: inline-block;
+    line-height: 1.5625rem;
+    padding-top: 6px;
+  }
+
+  & > h1 > span > a {
+    color: #000;
+    text-decoration: none;
+    line-height: 2.1875rem;
```

Comment:
Where is this value based on, is it a default?

Reply:
removed